clang/docs/ReleaseNotes.rst

```diff
@@ -481,6 +481,7 @@ Bug Fixes in This Version
   instantiated in one module and whose definition is instantiated in another
   module may end up with members associated with the wrong declaration of the
   class, which can result in miscompiles in some cases.
+<<<<<<< HEAD
```

Comment:
This seems like a error?

Reply:
Well that's weird... I thought I resolved all the merge conflicts before pushing. I'll fix that up.